start-site/src/main/resources/application.yml

```diff
@@ -867,6 +867,8 @@ initializr:
               version: 3.20.6
             - compatibilityRange: "[3.0.0-M1,3.1.0-M1)"
               version: 4.0.0-M3
+            - compatibilityRange: "[3.1.0,3.2.0-M1)"
```

Comment:
To avoid a gap in the version ranges, this should use 3.1.0-M1 as the lower bound.